src/components/MonthlyBreakdownTable.tsx

```
@@ -19,22 +23,66 @@ interface MonthlyBreakdownTableProps {
 type MonthlyRow = MonthlyData & { profitChangePercent: number | null };
 
+// Helper hook to fetch revenue breakdown for all months at once
+const useMonthlyRevenueBreakdowns = (
+  restaurantId: string | null,
+  months: string[]
+) => {
+  // Fetch all month breakdowns in parallel
+  const breakdowns = months.map(monthStr => {
+    const monthDate = parse(monthStr, 'yyyy-MM', new Date());
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    const { data } = useRevenueBreakdown(
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)